### PR TITLE
Template update for nf-core/tools version 2.13.1

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/sare
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
 - [ ] Make sure your code lints (`nf-core lint`).
-- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
+- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
```
Comment (asp8200) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: Where is one supposed to run that command?  In the project root dir?

Comment (maxulysse) [reply .github/PULL_REQUEST_TEMPLATE.md:21]: good point, that's not well done

Comment (maxulysse) [reply .github/PULL_REQUEST_TEMPLATE.md:21]: and the command won't work

Comment (maxulysse) [reply .github/PULL_REQUEST_TEMPLATE.md:21]:
```suggestion
- [ ] Ensure the test suite passes (`nf-test test tests/ --profile +docker`).
```

Comment (maxulysse) [reply .github/PULL_REQUEST_TEMPLATE.md]: cf https://github.com/nf-core/tools/pull/2821

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/sare
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
 - [ ] Make sure your code lints (`nf-core lint`).
-- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
+- [ ] Ensure the test suite passes (`nf-test test tests/ --verbose --debug --profile +docker`).
```
Comment (asp8200) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: That test cmd worked fine for me 🙂

Comment (maxulysse) [reply .github/PULL_REQUEST_TEMPLATE.md:21]: Toppen

**Review by FriederikeHanssen — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (maxulysse) [reply .github/PULL_REQUEST_TEMPLATE.md:21]:
```suggestion
- [ ] Ensure the test suite passes (`nf-test test tests/ --verbose --profile +docker`).
```

**Pull request merged**